src/rdb/tests/rdb_test_runner.py

```diff
@@ -1,288 +1 @@
-               daos_server))
-    except Exception as e:
-        if daos_server is None:
-            print("No PID was found for the DAOS server")
-        elif "No such process" in e:
-            print("The daos_server process is no longer available"
-                  " and could not be killed.")
-        else:
-            print("Unable to shut down DAOS server: {}".format(e))
-
-if __name__ == "__main__":
-    """
-    Start a DAOS server and then run the four stages of the client.
-    """
-    print("Running rdb tests")
-    rc = 0
-    binfo = BuildInfo(os.path.join(build_root, ".build_vars.json"));
-    debug_cmds = "-x D_LOG_MASK=DEBUG,RPC=ERR,MEM=ERR " + \
-                 "-x DD_SUBSYS=all -x DD_MASK=all"
-    if not load_mpi('openmpi'):
-        raise ServerFailedToStart("No orterun installed")
-    orterun = find_executable('orterun')
-    if orterun is None:
-        raise ServerFailedToStart("No orterun installed")
-
-    try:
-        # Server operations
-        p = start_server(binfo, orterun)
-
-        counter = 0
-        daos_server = daos_server_pid()
-        while daos_server is None:
-            if counter >= 120:
-                raise ServerTimedOut("No DAOS server process detected before "\
-                                     "timeout")
-            counter += 1
-            time.sleep(1)
-            daos_server = daos_server_pid()
-
-        # Give daos_engine some time to get ready.
-        time.sleep(10)
-
-        print("DAOS server started")
-
-        # Client operations
-        client_prefix = "{} --ompi-server " \
-                        "file:{} {} --np 1 rdbt ".format(
-                        orterun urifile, debug_cmds)
-        client_suffix = " --group=daos_server"
-        # orterun is called for the client four times: init, update, test,
-        # and fini
-        client_segments = ['init', 'update', 'test', 'fini']
-
-        try:
-            for segment in client_segments:
-                run_client(segment)
-            print("SUCCESS\nrbd tests PASSED")
-        except Exception as e:
-            print("rbd tests FAILED")
-            print("{}".format(e))
-            rc = 1
-
-    except ServerFailedToStart as e:
-        print("ServerFailedToStart: {}".format(e.message))
-        print("FAIL")
-        rc = 1
-
-    except ServerTimedOut as e:
-        print("ServerTimedOut: {}".format(e))
-        print("FAIL")
-        rc = 1
-
-    finally:
-        # Shut down the DAOS server when we are finished.
-        try:
-            if not p or p.poll() is not None:
-                # If the server is dead, something went very wrong
-                print("The server is unexpectedly absent.")
-                print("FAIL")
-                rc = 1
-        except NameError:
-            rc = 1
-        try:
-            cleanup(daos_server)
-        except NameError:
-            # The daos_server was never defined.
-            rc = 1
-
-    sys.exit(rc)
+Non
```

Comment:
(pylint-missing-docstring) Missing module docstring
(pylint-pointless-statement) Statement seems to have no effect
(pylint-undefined-variable) Undefined variable 'Non'